cinema/serializers.py

```diff
@@ -79,6 +106,7 @@
             "cinema_hall_name",
             "cinema_hall_capacity",
             "tickets_available",
+            "movie_image",
```

Comment:
Code Efficiency Issue: You've correctly used `serializers.ImageField` for the `movie_image` field in the `MovieSessionListSerializer`. Good job on this!